clientlib/src/main/proto/yelp/nrtsearch/luceneserver.proto

```diff
@@ -538,6 +538,7 @@ message StartIndexRequest {
     string primaryAddress = 4; //replica, the IP address or host name of the remote primary
     int32 port = 5; //replica, the TCP port of the remote primary
     RestoreIndex restore = 6; // restore index from backup
+    bool disableV0Archiver = 7; //if set to true; use IndexArchiver (IncrementalArchiver)
```

Comment:
This option here allows us to control starting indexing without having to restart the process. 

Also, we could potentially have same index restored using both old and new archiver (if also backed up with those names using same flag) on the same host if needed this way (e.g. for testing).

Reply:
This makes it harder to deploy this change since we would have to make changes in the operator. With a config file parameter we won't need to make any changes on operator side.

Reply:
As discussed in person will `restoreFromIncArchiver` and `backupWithIncArchiver` config flags to trigger usage of IncArchiver